Week1/4th_assignment.html

```diff
@@ -1,15 +1,149 @@
-    <main></main>
-    <footer></footer>
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="4th_assginment.css" />
+    <script
+      src="https://kit.fontawesome.com/67f5caa03d.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <h1 class="header_title">WEB TO DO MATE</h1>
+    </header>
+    <main>
+      <section class="weekly">
+        <!--월-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">월</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">27</p>
+        </div>
+        <!--화-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">화</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">28</p>
+        </div>
+        <!--수-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">수</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">29</p>
+        </div>
+        <!--목-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">목</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart" style="color: aqua"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">30</p>
+        </div>
+        <!--금-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">금</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">31</p>
+        </div>
+        <!--토-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">토</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">9</p>
+          </div>
+          <p class="weekly_datenum">1</p>
+        </div>
+        <!--일-->
+        <div class="weekly_date">
+          <h4 class="weekly_day">일</h4>
+          <div class="weekly_icon">
+            <i class="fa-solid fa-heart"></i>
+            <p class="weekly_todonum">6</p>
+          </div>
+          <p class="weekly_datenum">2</p>
+        </div>
+      </section>
+      <section class="todo">
+        <!--1번째 리스트-->
+        <article class="todo_list">
+          <div class="list_title" style="background-color: burlywood">
+            <h3 class="list_title_name">Wanna do</h3>
+            <button><i class="fa-solid fa-circle-plus"></i></button>
+          </div>
+          <p class="list_content">
+            <i class="fa-solid fa-heart" style="color: pink"></i> 테라스에서
+            커피마시기
+          </p>
```

Comment:
```suggestion
 <p class="list_content">
            <i class="fa-solid fa-heart" style="color: pink"></i> 테라스에서
            <span>커피마시기</span>
          </p>
```
으로 하고 .list_content 에 display: flex, align-items:center 하면 정렬델것가튼디!

Reply:
`div를 하나 만들고 태그와 아이콘을 따로 넣은 뒤에 flex 부여하고 align-item을 center줬어야 했는데 한번 이렇게 만들었더니 고치는 법을 도통 모르겠습니다. `요거에 대한 내 의견!